File: _persist/temp/venda/CSharp/Transitem.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("TRANSITEM")]
    public class Transitem : CollectionItem
    {
        [Campo("ID_TRANSACAO", CampoTipo.tfKey)]
        public string Id_Transacao { get; set; }
        [Campo("NR_ITEM", CampoTipo.tfKey)]
        public int Nr_Item { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("ID_PRODUTO", CampoTipo.tfReq)]
        public string Id_Produto { get; set; }
        [Campo("CD_PRODUTO", CampoTipo.tfReq)]
        public int Cd_Produto { get; set; }
        [Campo("DS_PRODUTO", CampoTipo.tfReq)]
        public string Ds_Produto { get; set; }
        [Campo("CD_CFOP", CampoTipo.tfReq)]
        public int Cd_Cfop { get; set; }
        [Campo("CD_ESPECIE", CampoTipo.tfReq)]
        public string Cd_Especie { get; set; }
        [Campo("CD_NCM", CampoTipo.tfReq)]
        public string Cd_Ncm { get; set; }
        [Campo("QT_ITEM", CampoTipo.tfReq)]
        public double Qt_Item { get; set; }
        [Campo("VL_CUSTO", CampoTipo.tfReq)]
        public double Vl_Custo { get; set; }
        [Campo("VL_UNITARIO", CampoTipo.tfReq)]
        public double Vl_Unitario { get; set; }
        [Campo("VL_ITEM", CampoTipo.tfReq)]
        public double Vl_Item { get; set; }
        [Campo("VL_VARIACAO", CampoTipo.tfReq)]
        public double Vl_Variacao { get; set; }
        [Campo("VL_VARIACAOCAPA", CampoTipo.tfReq)]
        public double Vl_Variacaocapa { get; set; }
        [Campo("VL_FRETE", CampoTipo.tfReq)]
        public double Vl_Frete { get; set; }
        [Campo("VL_SEGURO", CampoTipo.tfReq)]
        public double Vl_Seguro { get; set; }
        [Campo("VL_OUTRO", CampoTipo.tfReq)]
        public double Vl_Outro { get; set; }
        [Campo("VL_DESPESA", CampoTipo.tfReq)]
        public double Vl_Despesa { get; set; }
    }
}

File: _persist/temp/venda/Java/Transitem.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("TRANSITEM")
public class Transitem extends CollectionItem
{
    @Campo("ID_TRANSACAO", CampoTipo.tfKey)
    public String Id_Transacao;
    @Campo("NR_ITEM", CampoTipo.tfKey)
    public Integer Nr_Item;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("ID_PRODUTO", CampoTipo.tfReq)
    public String Id_Produto;
    @Campo("CD_PRODUTO", CampoTipo.tfReq)
    public Integer Cd_Produto;
    @Campo("DS_PRODUTO", CampoTipo.tfReq)
    public String Ds_Produto;
    @Campo("CD_CFOP", CampoTipo.tfReq)
    public Integer Cd_Cfop;
    @Campo("CD_ESPECIE", CampoTipo.tfReq)
    public String Cd_Especie;
    @Campo("CD_NCM", CampoTipo.tfReq)
    public String Cd_Ncm;
    @Campo("QT_ITEM", CampoTipo.tfReq)
    public Float Qt_Item;
    @Campo("VL_CUSTO", CampoTipo.tfReq)
    public Float Vl_Custo;
    @Campo("VL_UNITARIO", CampoTipo.tfReq)
    public Float Vl_Unitario;
    @Campo("VL_ITEM", CampoTipo.tfReq)
    public Float Vl_Item;
    @Campo("VL_VARIACAO", CampoTipo.tfReq)
    public Float Vl_Variacao;
    @Campo("VL_VARIACAOCAPA", CampoTipo.tfReq)
    public Float Vl_Variacaocapa;
    @Campo("VL_FRETE", CampoTipo.tfReq)
    public Float Vl_Frete;
    @Campo("VL_SEGURO", CampoTipo.tfReq)
    public Float Vl_Seguro;
    @Campo("VL_OUTRO", CampoTipo.tfReq)
    public Float Vl_Outro;
    @Campo("VL_DESPESA", CampoTipo.tfReq)
    public Float Vl_Despesa;
}